src/main/java/net/ripe/ipresource/IpAddress.java

```diff
@@ -40,9 +40,10 @@ public static IpAddress parse(String s) {
     }
 
     public static IpAddress parse(String s, boolean defaultMissingOctets) {
-        if (findFirstDotOrColon(s) == '.') {
+        final char findFirstDotOrColonResult = findFirstDotOrColon(s);
```

Comment:
s/findFirstDotOrColonResult/firstDotOrColonResult/g ?

Reply:
I am not sure to understand. Do you want to rename the new local variable I introduced to `findFirstDotOrColon`?

Reply:
I think he meant to rename the variable`firstDotOrColon` - at least, that would be clearer for me.

Reply:
OK, thanks. The last commit I made fixes this.

Reply:
Yes, exactly that. Thanks!